### ISSUE TITLE
Any plans to release a paid version with no crack time limit?

### ISSUE THREAD

**Issue by pyoseong** [km2ii/JVP_TG-PLUS#187]
Any plans to release a paid version with no crack time limit?

**Comment by mark8957** [comment]
agree

**Comment by teijiIshida** [comment]
Yes, many of us do want a full version with no restrictions for a reasonable price.

**Comment by newborn2010** [comment]
+1

**Comment by vu84b4** [comment]
+1

**Comment by georgechoo1990** [comment]
+1

**Comment by DeathPayner** [comment]
+1

**Comment by vu84b4** [comment]
+1

**Comment by PatWong0308** [comment]
To reduce private copies and prevent commercial use,I agree that the paid version should (1) include unit embedded code and the video generated will also embedded with this code. Only the author can identify those pirated/commercial copies. (2) limited time patch will be sent automatically to users. The patch will ban those illegal copies from updating.

**Comment by lfeng35** [comment]
I really need a paid full version. Love tg-plus but 50 limit is super annoying.
